src/main/kotlin/model/VCD.kt

```diff
@@ -0,0 +1,77 @@
+package uk.gov.justice.laa.architecture
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.Person
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ContainerView
+import com.structurizr.view.ViewSet
+
+class VCD private constructor() {
+  companion object : LAASoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var web: Container
+    lateinit var applicationCaseWorker: Person
+    lateinit var billingCaseWorker: Person
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "VCD (View Court Data)", 
```

Comment:
Shall we try to avoid acronyms in the actual diagrams? I'm open to both though.

```suggestion
        "View Court Data", 
```